.eslintrc

```diff
@@ -1,14 +1,31 @@
 {
-  "extends": ["wesbos"],
+  "root": true,
   "globals": {
     "fathom": "readonly"
   },
+  "env": {
+    "node": true,
+    "browser": true,
+    "commonjs": true,
+    "es6": true
+  },
+  "parser": "babel-eslint",
+  "extends": [
```

Comment:
Would it be helpful to try out: https://www.npmjs.com/package/eslint-config-carbon ? Might help with some of the base rules and then can layer on the import ones 👀

Reply:
@joshblack for sure, can we update the peer dep to allow eslint 7?

Reply:
Hell yeah 🔥